base_address_parish/views/res_parish_view.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+        <record id="view_parish_tree" model="ir.ui.view">
+            <field name="model">res.parish</field>
+            <field name="arch" type="xml">
+                <tree editable="top">
+                    <field name="name" />
+                    <field name="zip" />
+                    <field name="city_id" context="{'default_state_id': state_id}" />
+                    <field name="state_id" />
+                    <field name="country_id" />
+
+                </tree>
+            </field>
+        </record>
+
+        <record id="view_parish_filter" model="ir.ui.view">
+            <field name="model">res.parish</field>
+            <field name="arch" type="xml">
+                <search string="Search Parish">
+                    <field
+                    name="name"
+                    filter_domain="['|', ('name','ilike',self), ('zip','ilike',self)]"
+                    string="Parish"
+                />
+                    <separator />
+                    <field name="country_id" />
+                    <field name="state_id" />
+                    <field name="city_id" />
+                </search>
+            </field>
+        </record>
+
+        <record id="action_res_parish_tree" model="ir.actions.act_window">
+            <field name="name">Parishes</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">res.parish</field>
+            <field name="view_mode">tree</field>
+            <field name="help">
+                Display and manage the list of all parishes that can be assigned to
+                your partner records. Note that an option can be set on each country separately
+                to enforce any address of it to have a parish in this list.
+            </field>
+        </record>
+
+        <!--MENU ITEMS FOR CITIES-->
+        <menuitem
+        name="Cities"
+        id="locations_menu_cities"
+        parent="contacts.menu_localisation"
+        action="base_address_city.action_res_city_tree"
+        sequence="4"
+    />
```

Comment:
I see menu is added on other module([base_location ](https://github.com/OCA/partner-contact/blob/15.0/base_location/views/res_city_view.xml#L53)) and see two menus. I think this module should not add menu

![image](https://user-images.githubusercontent.com/7775116/155399154-f8e83396-ec01-4b77-aa21-be45564ef0ad.png)

Reply:
Solved